<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IsoSDKDataSample.qrc">
    <normaloff>:/new/prefix1/images/isosdk.ico</normaloff>:/new/prefix1/images/isosdk.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Burn device: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbBurnDrive"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Disc file system:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbProject"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkImportLastSession">
        <property name="text">
         <string>Import last session</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkUsePath">
        <property name="text">
         <string>Save original path on disk</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkISOLevel3">
        <property name="text">
         <string>Use ISO Level 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="totalProgress">
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuIngo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionAdd_Folder_2"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionBurn"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionEject_Disk_after_burn"/>
    <addaction name="actionUser_Buffer_Protection"/>
    <addaction name="actionEraseRW_before_burn"/>
   </widget>
   <addaction name="menuIngo"/>
   <addaction name="menuAction"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionReset"/>
   <addaction name="separator"/>
   <addaction name="actionBurn"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="IsoSDKDataSample.qrc">
     <normaloff>:/new/prefix1/images/fileadd.png</normaloff>:/new/prefix1/images/fileadd.png</iconset>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="IsoSDKDataSample.qrc">
     <normaloff>:/new/prefix1/images/deleteall.png</normaloff>:/new/prefix1/images/deleteall.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionBurn">
   <property name="icon">
    <iconset resource="IsoSDKDataSample.qrc">
     <normaloff>:/new/prefix1/images/burn_32.png</normaloff>:/new/prefix1/images/burn_32.png</iconset>
   </property>
   <property name="text">
    <string>Burn</string>
   </property>
  </action>
  <action name="actionEject_Disk_after_burn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eject Disk after burn</string>
   </property>
  </action>
  <action name="actionUser_Buffer_Protection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>User Buffer  Protection</string>
   </property>
  </action>
  <action name="actionEraseRW_before_burn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EraseRW before burn</string>
   </property>
  </action>
  <action name="actionAdd_Folder">
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
  <action name="actionAdd_Folder_2">
   <property name="icon">
    <iconset resource="IsoSDKDataSample.qrc">
     <normaloff>:/new/prefix1/images/folder_add32.png</normaloff>:/new/prefix1/images/folder_add32.png</iconset>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IsoSDKDataSample.qrc"/>
 </resources>
 <connections/>
</ui>
